.tutone.yml

```diff
@@ -875,6 +875,9 @@ packages:
         field_type_override: users.UserReference
         skip_type_create: true
 
+      - name: AgentApplicationSettingsRawJsConfiguration
+        create_as: map[string]interface{}
+
```

Comment:
Thank you for helping identify this, @sanderblue 🙂 

> Thanks for adding this comment. This will help future devs know this was manually added when they see the diff. 
> 
> That being said, I _think_ you should be able to override its type generation in the `.tutone.yml` config file by adding the following to the entities package[ `types`](https://github.com/newrelic/newrelic-client-go/blob/main/.tutone.yml#L647) config:
> ```yaml
> - name: AgentApplicationSettingsRawJsConfiguration
>   create_as: "map[string]interface{}"
> ```

(To verify if this change works as intended, after making this change, I tried a `tutone generate --refetch --package entities` which did not change the type of `AgentApplicationSettingsRawJsConfiguration`).